core-bundle/src/Resources/config/services.yml

```diff
@@ -455,7 +455,7 @@ services:
         class: Contao\CoreBundle\Monolog\ContaoTableHandler
         arguments:
             - debug
-            - false
+            - true
```

Comment:
I think we can drop the whole `arguments:` section now, because we are using the defaults now.

Reply:
Done in d6c303a46f43c60fda77d56c25029d0df40d08f1